docker-compose.yml

```
@@ -38,6 +38,32 @@ services:
       interval: 5s
       retries: 20
 
+  # applies OpenFGA's database schema migrations
+  openfga-migrate:
+    depends_on:
+      postgres:
+        condition: service_healthy
+    image: openfga/openfga:latest
+    container_name: osrd-openfga-migrate
+    command: migrate
+    environment:
+      OPENFGA_DATASTORE_ENGINE: "postgres"
+      OPENFGA_DATASTORE_URI: "postgres://osrd:password@postgres:5432/osrd"
+
+  openfga:
+    depends_on:
+      openfga-migrate:
+        condition: service_completed_successfully
+    image: openfga/openfga:latest
+    container_name: osrd-openfga
+    environment:
+      OPENFGA_DATASTORE_ENGINE: "postgres"
+      OPENFGA_DATASTORE_URI: "postgres://osrd:password@postgres:5432/osrd"
```

We should use another database.

```suggestion
      OPENFGA_DATASTORE_URI: "postgres://osrd:password@postgres:5432/openfga"
```